### PR TITLE
feat: hero video

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: components/homepage/HowItWorksBlock.tsx [hunk 1]
```diff
@@ -148,7 +148,7 @@ const HowItWorksBlock = (props: any) => {
                             [`@media screen and (min-width: ${theme.breakpoints[4]})`]:
                                 {
                                     width: "57.6rem !important",
-                                    py: "7.4rem !important",
+                                    // py: "7.4rem !important",
```
Comment (guerrap) [inline components/homepage/HowItWorksBlock.tsx:151]: Should we keep this comment?

File: components/homepage/Hero.tsx [hunk 2]
```diff
@@ -136,39 +135,37 @@ const Hero = () => {
...
                                     "31.2rem",
                                     "40.8rem",
                                 ],
-                                height: [
-                                    "120vw",
-                                    "calc(2.4rem * 16)",
-                                    "43.66rem",
-                                    "57rem",
-                                ],
+                                // paddingTop: "56.25%",
```
Comment (guerrap) [inline components/homepage/Hero.tsx:146]: Comment to remove.

File: components/homepage/HowItWorksBlock.tsx (hunk 1 above)
Comment (luzzif) [reply components/homepage/HowItWorksBlock.tsx:151]: Addressed in 00ab6e73c6e7d0a5fc453fc39db06032ab7b4b39

File: components/homepage/Hero.tsx (hunk 2 above)
Comment (luzzif) [reply components/homepage/Hero.tsx:146]: Addressed in 00ab6e73c6e7d0a5fc453fc39db06032ab7b4b39

File: components/homepage/HowItWorksBlock.tsx (hunk 1 above)
Comment (luzzif) [reply components/homepage/HowItWorksBlock.tsx:151]: Addressed in 00ab6e73c6e7d0a5fc453fc39db06032ab7b4b39

File: components/homepage/Hero.tsx (hunk 2 above)
Comment (luzzif) [reply components/homepage/Hero.tsx]: Addressed in 00ab6e73c6e7d0a5fc453fc39db06032ab7b4b39

**Review by guerrap — APPROVED** [review]

**Review by guerrap — APPROVED** [review]

**Pull request merged**